src/api/article/localized/controller.js

```diff
@@ -0,0 +1,31 @@
+import { checkIsFound } from 'utils/validation';
+import { sendJson } from 'utils/api';
+
+import Article from 'api/article/article.model';
+
+import LocalizedArticle from './model';
+
+export const create = async ({ params: { articleId }, body }, res, next) =>
+  Article.findOne({ _id: articleId })
+    .then(checkIsFound)
+    .then(article => {
+      LocalizedArticle({ ...body, articleId: article._id })
+        .save()
+        .then(data => {
+          article.locales.push(data._id);
+          return data;
+        })
+        .then(data => {
+          article.save();
+          return data;
+        })
+        .then(sendJson(res))
+        .catch(next);
+    })
+    .catch(next);
```

Comment:
```js
async create(...) => {
	try {
		const article = await Article.findOne({ _id: articleId });
		checkIsFound(article);
	} catch err {
}
```

Reply:
it's probably more readable but I still not sure when and why do we want to rewrite such structure into a chain from try-catch or back.
I think it's fine as it's written already.